policy-service/src/contract-tests/java/uk/gov/gchq/palisade/contract/policy/kafka/KafkaContractTest.java

```diff
@@ -171,20 +171,24 @@ void testPolicyExistsForValidRequest() {
         // All messages have a correct Token in the header
         assertAll("Headers have correct token",
                 () -> assertThat(results)
+                        .as("Check the size of the returned results")
                         .hasSize((int) recordCount),
 
                 () -> assertThat(results)
                         .allSatisfy(result ->
```

Comment:
Only one assertion, do not need allSatisfy

Reply:
This is making sure that all the values within the `results` list have the expected token value